src/components/Nav.jsx

```diff
@@ -0,0 +1,9 @@
+import { Link } from "react-router-dom"
+
+function Nav (){
+
+    return (
+    <Link to="/reviews">go to reviews</Link>
+    )
```

Comment:
this link works, but looks a bit out of place between the header and article... You might consider making your header a link home to improve how the site looks. Optional design choice - not a dealbreaker